docs/upgrade-guide.md

```diff
@@ -8,6 +8,11 @@ The module entry point is now only lightly transpiled for the most commonly used
 
 ### Layers
 
+- A bug is fixed in projecting sizes in billboard mode. Pixel sizes now match their CSS counterparts. This change affects the following layers when used with a `MapView`:
+  + `ArcLayer`, `LineLayer` and `PointCloudLayer`
+  + `IconLayer` and `TextLayer` with the default `billboard` prop
+  + `PathLayer` with `billboard: true`
+  After upgrading to v8.5, in order to maintain the same appearance, you need to multiply `2/3` to the objects' width/size. This can be done by either changing the accessor (`getWidth`/`getSize`) or the scaling prop (`sizeScle`/`widthScale`).
```

Comment:
```suggestion
  After upgrading to v8.5, in order to maintain the same appearance, you need to multiply `2/3` to the objects' width/size. This can be done by either changing the accessor (`getWidth`/`getSize`) or the scaling prop (`sizeScale`/`widthScale`).
```